src/App.tsx

```diff
@@ -55,47 +58,64 @@ const queryClient = new QueryClient({
 });
 addDefaultMutationFns(queryClient);
 
-const store = new Storage();
-await store.create();
+const App: React.FC = () => {
+  const [persister, setPersister] = useState<Persister | null>(null);
 
-const persister = createAsyncStoragePersister({
-  storage: {
-    getItem: async (key) => store.get(key),
-    setItem: async (key, value) => store.set(key, value),
-    removeItem: async (key) => store.remove(key),
-  },
-});
+  useEffect(() => {
+    async function initPersister() {
+      const store = new Storage();
+      await store.create();
+      setPersister(
```

Comment:
Note to self: 

React components re-render when either (a) their props change or (b) their state changes. Since this `setPersister` call will change the component's state, it will cause a re-render; which will cause the...

```ts
  if (persister == null) {
    return null;
  }
```

...check to be skipped and for the "normal" element tree...

```ts
  return (
    <PersistQueryClientProvider
```

...to be rendered.